spec/controllers/application_controller_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe ApplicationController do
   describe "#report_only" do
-    let(:group) { FactoryBot.create(:miq_group, :miq_user_role => FactoryBot.create(:miq_user_role)) }
+    let(:group) { FactoryBot.create(:miq_group, :miq_user_role => FactoryBot.create(:miq_user_role, :features => %w[everything])) }
```

Comment:
Similarly this feels like we should have a factory for an "admin group with an admin user role"